docs/user-guide/horizontal-pod-autoscaling/index.md

```diff
@@ -79,55 +103,35 @@ i.e. you cannot bind a Horizontal Pod Autoscaler to a replication controller and
-  be placed in `definition.json` mounted via a [configMap](/docs/user-guide/configmap/) in `/etc/custom-metrics`. A sample config map may look like this:
-
-```yaml
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: cm-config
-data:
-  definition.json: "{\"endpoint\" : \"http://localhost:8080/metrics\"}"
-``` 
-
-**Warning**
-Due to the way cAdvisor currently works `localhost` refers to the node itself, not to the running pod. Thus the appropriate container in the pod must ask for a node port. Example:
-
-```yaml
-    ports:
-    - hostPort: 8080
-      containerPort: 8080
-```
-
-### Specifying target
-
-HPA for custom metrics is configured via an annotation. The value in the annotation is interpreted as a target metric value averaged over
-all running pods. Example: 
-
-```yaml
-    annotations:
-      alpha/target.custom-metrics.podautoscaler.kubernetes.io: '{"items":[{"name":"qps", "value": "10"}]}'
-```
+## Support for custom metrics
 
-In this case, if there are four pods running and each pod reports a QPS metric of 15 or higher, horizontal pod autoscaling will start two additional pods (for a total of six pods running).
+**Note**: Kubernetes 1.2 added alpha support for scaling based on application-specific metrics using special annotations.
+Support for this was removed in Kubernetes 1.6 in favor of the `autoscaling/v2alpha1` API.  While the old method for collecting
+custom metrics is still available, these metrics will not be available for use by the Horizontal Pod Autoscaler, and the former
+annotations for specifying which custom metrics to scale on are no longer honored by the Horizontal Pod Autoscaler controller.
 
-If you specify multiple metrics in your annotation or if you set a target CPU utilization, horizontal pod autoscaling will scale to according to the metric that requires the highest number of replicas.
+Kubernetes 1.6 adds supports for making use of custom metrics in the Horizontal Pod Autoscaler.
+You can custom metrics for the Horizontal Pod Autoscaler to use in the `autoscaling/v2alpha1` API.
+Kubernetes then queries the new custom metrics API to fetch the values of the appropriate custom metrics.
 
-If you do not specify a target for CPU utilization, Kubernetes defaults to an 80% utilization threshold for horizontal pod autoscaling.
+### Prerequisites
 
-If you want to ensure that horizontal pod autoscaling calculates the number of required replicas based only on custom metrics, you should set the CPU utilization target to a very large value (such as 100000%). As this level of CPU utilization isn't possible, horizontal pod autoscaling will calculate based only on the custom metrics (and min/max limits).
+In order to use custom metrics in the Horizontal Pod Autoscaler, you must have deployed the cluster with the
```

Comment:
Avoid past tense.

"To use custom metrics in the Horizontal Pod Autoscaler, you must deploy your cluster with the.."